src/tribler/gui/upgrade_manager.py

```diff
@@ -22,6 +22,7 @@
 
 class StateDirUpgradeWorker(QObject):
     finished = pyqtSignal(object)
+    cancelled = pyqtSignal(object)
```

Comment:
Question: should it be `cancelled = pyqtSignal(str)` instead?

Reply:
Good point :+1: